### DISCUSSION TITLE
Put seconds bar on main monitor

### DISCUSSION THREAD

**Discussion by Jorgebran** [martinet101/ElevenClock#477] [Q&A] [answered]
Hi!! Martine help I have two monitors, but I can't use ElevenClock on the main monitor, it always stays on the secondary monitor. The feature to show the seconds in ElevenClock is important to me, but it only gets the secondary monitor. In the clock settings there is show clock on the main screen, I understand that this option says that it is to show the Elevenclock with the seconds option on the main monitor, but that doesn't happen, I use the latest version 3.0.0. Is there anything I can do? 

Thank you very much, congratulations on the program!!!

**Comment by marticliment** [comment]
Can you please enable the first monitor thing and send me the log? (ElevenClock Settings -> Debugging tools)

**Comment by Jorgebran** [comment]
---------------------------------------------------------------------------------------------------


ElevenClock's 3.0.0 (v3) log: Select all the text and hit Ctrl+C to copy it

All modules loaded successfully and sys.stdout patched correctly, starting
main script

Translator function set language to "pt_BR"


---------------------------------------------------------------------------------------------------


Log legend:

🔵: Verbose

🟢: Information

🟡: Expected warning

🟠: Unexpected warning

🔴: Error


🔵 Initializing clock 0...

🔵 Using taskbar background color

🟢 Regular sized taskbar

🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 1920, 1080)

🟡 Taskbar at bottom

🟡 Clock on the left

🟢 Using qt's default positioning system

🔵 Clock geometry: PySide2.QtCore.QRect(8, 1032, 200, 48)

🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']

🔵 Font size: 9.3

🟢 Using black text (light mode)

🔵 Full screen rect: (0, 0, 1920, 1080)

🔵 Show/hide loop started with parameters: HideonFS:False,
NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:False

🟡 Desktop button enabled

🔵 Restart loop: 1

🔵 Initializing clock 1...

🔵 Using taskbar background color

🟢 Regular sized taskbar

🔵 Monitor geometry: PySide2.QtCore.QRect(1920, 0, 1920, 1080)

🟡 Taskbar at bottom

🟡 Clock on the left

🟢 Using qt's default positioning system

🔵 Clock geometry: PySide2.QtCore.QRect(1928, 1032, 200, 48)

🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']

🔵 Font size: 9.3

🟢 Using black text (light mode)

🔵 Full screen rect: (1920, 0, 3840, 1080)

🔵 Show/hide loop started with parameters: HideonFS:False,
NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:False

🟡 Desktop button enabled

🟢 Loaded clocks in 1.7813751697540283

🔵 System tray icon enabled

🔵 Starting update check

🟢 Loaded everything in 2.2194933891296387

🔵 Version URL:
https://versions.somepythonthings.tk/versions/elevenclock.ver

🟢 Updates not found

🔵 Starting update check

🔴 Traceback (most recent call last):


🔴 File "urllib\request.py", line 1346, in do_open


🔴 File "http\client.py", line 1285, in request


🔴 File "http\client.py", line 1331, in _send_request


🔴 File "http\client.py", line 1280, in endheaders


🔴 File "http\client.py", line 1040, in _send_output


🔴 File "http\client.py", line 980, in send


🔴 File "http\client.py", line 1454, in connect


🔴 File "ssl.py", line 500, in wrap_socket


🔴 File "ssl.py", line 1040, in _create


🔴 File "ssl.py", line 1309, in do_handshake


🔴 FileNotFoundError: [Errno 2] No such file or directory


🔴

During handling of the above exception, another exception occurred:



🔴 Traceback (most recent call last):


🔴 File "\\samba-server\SPTPrograms\ElevenClock\elevenclock\__init__.py",
line 121, in updateIfPossible


🔴 File "urllib\request.py", line 214, in urlopen


🔴 File "urllib\request.py", line 517, in open


🔴 File "urllib\request.py", line 534, in _open


🔴 File "urllib\request.py", line 494, in _call_chain


🔴 File "urllib\request.py", line 1389, in https_open


🔴 File "urllib\request.py", line 1349, in do_open


🔴 urllib.error.URLError: <urlopen error [Errno 2] No such file or
directory>


🔴 Note this traceback was caught by reporter and has been added to the log
(<urlopen error [Errno 2] No such file or directory>)

🔵 Version URL: http://www.somepythonthings.tk/versions/elevenclock.ver

🟢 Updates not found

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! Prince of Persia chega ao Apple Watch graças
a versão em JavaScript - TudoCelular.com - Google Chrome (1920, 0, 3840,
1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Fullscreen window detected! "É o Brian": Motorista é filmado acelerando
em marcha a ré em estrada no RS - 14/01/2022 - UOL Carros - Google Chrome
(1920, 0, 3840, 1080) Fullscreen rect: (1920, 0, 3840, 1080)

🟡 Closing clock on {'Monitor': (0, 0, 1920, 1080), 'Work': (0, 0, 1920,
1032), 'Flags': 1, 'Device': '\\\\.\\DISPLAY2'}

🟡 Closing clock on {'Monitor': (1920, 0, 3840, 1080), 'Work': (1920, 0,
3840, 1032), 'Flags': 0, 'Device': '\\\\.\\DISPLAY1'}

🔵 Initializing clock 0...

🔵 Using taskbar background color

🟢 Regular sized taskbar

🔵 Monitor geometry: PySide2.QtCore.QRect(0, 0, 1920, 1080)

🟡 Taskbar at bottom

🟡 Clock on the left

🟢 Using qt's default positioning system

🔵 Clock geometry: PySide2.QtCore.QRect(8, 1032, 200, 48)

🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']

🔵 Font size: 9.3

🟢 Using black text (light mode)

🔵 Full screen rect: (0, 0, 1920, 1080)

🔵 Show/hide loop started with parameters: HideonFS:False,
NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:False

🟡 Desktop button enabled

🔵 Initializing clock 1...

🔵 Using taskbar background color

🟢 Regular sized taskbar

🔵 Monitor geometry: PySide2.QtCore.QRect(1920, 0, 1920, 1080)

🟡 Taskbar at bottom

🟡 Clock on the left

🟢 Using qt's default positioning system

🔵 Clock geometry: PySide2.QtCore.QRect(1928, 1032, 200, 48)

🔵 Font families: ['Segoe UI Variable Display', 'sans-serif']

🔵 Font size: 9.3

🟢 Using black text (light mode)

🔵 Full screen rect: (1920, 0, 3840, 1080)

🔵 Show/hide loop started with parameters: HideonFS:False,
NotHideOnTB:False, HideOnRDP:False, ClockOn1Mon:True, NefWSMethod:False

🟡 Desktop button enabled

🔵 Restart loop: 1



Em sex., 14 de jan. de 2022 às 06:51, Martí Climent <
***@***.***> escreveu:

**Comment by marticliment** [comment]
Can you please detele the .elevenclock folder placed in your user directory?

**Comment by Jorgebran** [comment, marked as answer]
Thank you very much Martine. I've been using the app for a while, and then
just updated to new versions. I've uninstalled and reinstalled and it
didn't solve the problem. Had to uninstall the app and also that folder.
Only after that did it really catch on. Thanks

Em sex., 14 de jan. de 2022 às 12:17, Martí Climent <
***@***.***> escreveu:
